### ISSUE TITLE
When assertion count is incorrect, output the expected and actual values.

### ISSUE THREAD

**Issue by dylang** [sindresorhus/ava#102]
I see that the actual count is recorded, but I'm not sure how to see it when running `ava`.

https://github.com/sindresorhus/ava/blob/master/lib/test.js#L150-L158

```
❯ npm test

> npm-check@4.1.2 test /Users/dgreene/projects/npm-check
> xo && ava ./test


  ✖ require Assertion count does not match planned

  1 test failed

  1. require
  Error
    at Test.plan (/Users/dgreene/projects/npm-check/node_modules/ava/lib/test.js:87:19)
    at Test.<anonymous> (/Users/dgreene/projects/npm-check/test/test.js:10:7)
    at step (/Users/dgreene/projects/npm-check/test/test.js:5:273)
    at Test.fn (/Users/dgreene/projects/npm-check/test/test.js:5:99)
```

The above is using using `master` with `power-assert` added.

Similar output using the published version

```
> npm-check@4.1.2 test /Users/dgreene/projects/npm-check
> xo && ava ./test


  ✖ require Assertion count does not match planned

  1 test failed

  1. require
  Error
    at Test.plan (/Users/dgreene/projects/npm-check/node_modules/ava/lib/test.js:59:19)
    at Object.callee$0$0$ (/Users/dgreene/projects/npm-check/test/test.js:10:7)
    at Test.callee$0$0 [as fn] (/Users/dgreene/projects/npm-check/test/test.js:32:26)
```

**Comment by sindresorhus** [comment]
Confirmed.

**Comment by dylang** [comment]
It seems simple to change the message in https://github.com/sindresorhus/ava/blob/04b3db5d96d6530d6b03c5f8f4c915dea16cd125/lib/test.js#L155, or would you prefer to use the stored `actual` and `expected` properties in the rederer?

**Comment by sindresorhus** [comment]
The actual stored props should be used, but unsure why they're not. Maybe because we overwrite the stack in https://github.com/sindresorhus/ava/blob/04b3db5d96d6530d6b03c5f8f4c915dea16cd125/lib/test.js#L159.

**Comment by ariporad** [comment]
Ok, @sindresorhus: Here's the problem. Actual/expected are only included when no message is provided, see [here](https://github.com/sindresorhus/core-assert/blob/master/index.js#L68-L74) and [here](https://github.com/sindresorhus/core-assert/blob/master/index.js#L90-L94). So we have to include that in the error message ourself. I'll submit a PR.

**Comment by ariporad** [comment]
Fixed in #376.

**Comment by zweicoder** [comment]
+1, would like to see this fixed!
